cmd/allocator/main.go

```diff
@@ -66,10 +67,19 @@ const (
 func main() {
 	conf := parseEnvFlags()
 
-	logger.WithField("version", pkg.Version).WithField("ctlConf", conf).
+	logger.WithField("version", pkg.Version).WithField("conf", conf).
```

Comment:
I think it's a remnant of the controller.
I fixed it, but I'll put it back if it looks problematic.

Reply:
Yeah, can we roll this back. Better to be consistent between each binary.